ir/translation/constraints.rs

```diff
@@ -53,8 +57,26 @@ pub(super) fn add_statement(
             add_typeql_expression(function_index, constraints, assigned, rhs)?
         }
         typeql::Statement::Thing(thing) => add_thing_statement(function_index, constraints, thing)?,
-        typeql::Statement::AttributeValue(attribute_value) => todo!(),
-        typeql::Statement::AttributeComparison(_) => todo!(),
+        typeql::Statement::AttributeValue(attribute_value) => {
+            // TODO: Strip the expressions out once we have a ValueRegister.
+            let attribute = register_typeql_var(constraints, &attribute_value.var)?;
+            add_typeql_isa(constraints, attribute, &attribute_value.isa)?;
+            let rhs_var = constraints.create_anonymous_variable()?;
+            let mut rhs_value = ExpressionTree::empty();
+            let value = translate_literal(&attribute_value.value).map_err(|source| {
+                PatternDefinitionError::LiteralParseError { source, literal: attribute_value.value.to_string().clone() }
+            })?;
+            rhs_value.add(Expression::Constant(value));
+            constraints.add_expression(rhs_var, rhs_value)?;
+            constraints.add_comparison(attribute, rhs_var, Comparator::Equal)?;
+        }
+        typeql::Statement::AttributeComparison(attribute_comparison) => {
+            let attribute = register_typeql_var(constraints, &attribute_comparison.var)?;
+            add_typeql_isa(constraints, attribute, &attribute_comparison.isa)?;
+            let rhs_var = constraints.create_anonymous_variable()?;
+            add_typeql_expression(function_index, constraints, rhs_var, &attribute_comparison.comparison.rhs)?;
+            constraints.add_comparison(attribute, rhs_var, attribute_comparison.comparison.comparator.into())?;
+        }
```

Comment:
This might happen concurrently. I need to check with Dmitrii

Reply:
` // TODO: Strip the expressions out once we have a ValueRegister.`
^ This comment was about the TODO